scripts/docker/buildpack-deps/Dockerfile.ubuntu.clang.ossfuzz

```diff
@@ -70,22 +70,6 @@ RUN wget https://github.com/Kitware/CMake/releases/download/v3.21.2/cmake-3.21.2
 
 FROM base AS libraries
 
-# Boost
```

Comment:
[Base image](https://github.com/google/oss-fuzz/blob/master/projects/solidity/Dockerfile#L31-L43) already has boost - curious to see what happens if we get rid of it here.

Reply:
Guess not.